Makefile

```diff
@@ -27,6 +27,8 @@ BUILD_SUBDIRS := kernel runner third_party tools userspace
 # build/ directory.
 BWRAP := bwrap                                                               \
          --ro-bind / /                                                       \
+         --tmpfs /tmp                                                        \
```

Comment:
Can you please add a comment about this?

Reply:
Good point, done.